example/roblox/v2.yaml

```diff
@@ -0,0 +1,33 @@
+# normally this would be suffixed with the domain (.com)
+name: "cloud.roblox.com"
+url: "https://apis.roblox.com/cloud/v2"
+resources:
```

Comment:
@rofrankel FYI - I'm going to start adding more fields once aepcli gets some support for mutation operations. For now just the resource definition is all we need.